docs/applications/quick-start.md

```diff
@@ -143,144 +143,79 @@ export default NextAuth(NextAuthOptions);
-            👋
-          </Text>
-        </Box>
-        <Stack
-          direction="column"
-          m="auto"
-          spacing={3}
-          textAlign="left"
-          px={[0, 0, "10rem", "20rem"]}
-        >
-          <Box
-            w="full"
-            textAlign="left"
-            color="loginBtnGray"
-            fontFamily="inter"
-            fontWeight="normal"
-          >
-            Signup or Login with
-          </Box>
-          <HStack
-            onMouseEnter={() => setActiveBtn("google")}
-            onMouseLeave={() => setActiveBtn("")}
-          >
-            <Button
-              variant="login"
-              onClick={() => signInKeyp("GOOGLE")}
-              _hover={{
-                bg: "googleBlue",
-                color: "white",
-              }}
-            >
-              <Image
-                src={
-                  activeBtn === "google"
-                    ? "social-bg-white.svg"
-                    : "social-bg-light.svg"
-                }
-                alt=""
-                w="2.5rem"
-              />
-              <Box position="absolute" ml="0.65rem">
-                <FaGoogle
-                  color={activeBtn === "google" ? "#4285F4" : "white"}
-                  size="20px"
-                />
-              </Box>
-              <Text ml="1rem">Google</Text>
-            </Button>
-          </HStack>
-          <HStack
-            onMouseEnter={() => setActiveBtn("discord")}
-            onMouseLeave={() => setActiveBtn("")}
-          >
-            <Button
-              variant="login"
-              onClick={() => signInKeyp("DISCORD")}
-              _hover={{
-                bg: "discordBlue",
-                color: "white",
-              }}
-            >
-              <Image
-                src={
-                  activeBtn === "discord"
-                    ? "social-bg-white.svg"
-                    : "social-bg-light.svg"
-                }
-                alt=""
-                w="2.5rem"
-              />
-              <Box
-                position="absolute"
-                ml="0.65rem"
-                _hover={{
-                  color: "#5865F2",
-                }}
-              >
-                <FaDiscord
-                  color={activeBtn === "discord" ? "#5865F2" : "white"}
-                  size="20px"
-                />
-              </Box>
-              <Text ml="1rem">Discord</Text>
-            </Button>
-          </HStack>
-          <Box>
-            <Text color="#B0B6C1" fontSize="0.75rem" fontFamily="inter">
-              Powered by 🍩 Keyp
-            </Text>
-          </Box>
-        </Stack>
-      </Box>
-    </>
-  );
+    useEffect(() => {
+        if (session && session.status === "authenticated") {
+            router.push("/");
+        }
+    }, [session, router]);
+
+    const handleLoginClick = (provider: string) => {
+        signInKeyp(provider);
+    };
```

Comment:
I really like that you included this, rather than just passing `signInKep()` to `LoginPortal`. This makes it much easier to understand whats going on :)